images/proxy/build.sh

```diff
@@ -1,10 +1,21 @@
 #!/bin/bash
-# Build script
-set -eo pipefail
+
```

Comment:
@rjshrjndrn Add this line `set -eo pipefail`

Reply:
:+1: